src/main/java/io/fabric8/elasticsearch/plugin/kibana/KibanaSeed.java

```diff
@@ -407,10 +387,21 @@ private static void executeDelete(String index, String type, String id, Client e
 		
 		logger.debug("DELETE: '{}/{}/{}'", index, type, id);
 		
+		int timeout = 30,
+				count = 0;
+		
 		DeleteRequest request = esClient.prepareDelete(index, type, id).request();
 		try {
-			esClient.delete(request).get();
-		} catch (InterruptedException | ExecutionException e) {
+			ActionFuture<DeleteResponse> response = esClient.delete(request);
+			// wait until response isDone()
+			while ( !response.isDone() ) {
```

Comment:
I may be missing a plugin on my client... i'm seeing if fixing the client will let me revert to have `.get()` block instead of receiving a NPE